README.md

```diff
@@ -1,2 +1,94 @@
-# janitor
+<!-- omit from toc -->
+# Janitor
 A Python application which hosts task scripts for data management and clean up tasks, running them on a schedule.
+The application uses APScheduler to schedule and run these tasks.
```

Comment:
It would be nice to have if you could link `APScheduler` to the docs for that package.  Perhaps just the root of the repo.

Reply:
I've now added a link to the `APScheduler` documentation